Formula/howdoi.rb

```diff
@@ -3,8 +3,8 @@ class Howdoi < Formula
 
   desc "Instant coding answers via the command-line"
   homepage "https://github.com/gleitz/howdoi"
-  url "https://files.pythonhosted.org/packages/c9/4a/ddb9414149ada75e8b06e6810b1717bbd7439755e9e830cb7d53bb760805/howdoi-2.0.15.tar.gz"
-  sha256 "01181f9af1032f6d77e8574f231d4b0e3090d6f794c62f0f1eca900cacea89ce"
+  url "https://github.com/gleitz/howdoi/archive/refs/tags/v2.0.16.tar.gz"
+  sha256 "e32d6bce31f27862bf23007638a78ea6727ed325e863e3ea8e3b7b377f603bbf"
```

Comment:
Please use the PyPI URL instead.

Reply:
Fixed.

Reply:
I'm assuming you used `bump-formula-pr` with the `--url` argument, in this case just using `--version 2.0.16` would be fine.

Reply:
Indeed, first time using the tool, was not quite sure what to do ! Thanks for the tip.